docs/quickstarts/project-quickstart.md

```diff
@@ -92,15 +92,15 @@ Office.context.document.getSelectedTaskAsync(
 
 ## Try it out
 
-1. Start the local web server by running the following command:
+> [!NOTE]
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides.
```

Comment:
Good catch on the fact that the Mac note is not needed in this article!

Now that there's just one command to run, should change "one of the following commands" -> "the following command" in this note.

Reply:
```suggestion
> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
```